docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -93,23 +103,13 @@ Preparing For a Release
    The release may require certain changes to be back ported from the master branch.
    See :ref:`back-porting-changes`\ .
 
-#. Update the version numbers in:
-
-   .. note:: Skip this step for a documentation release.
-
-   - the ``pip install`` line in
-     `docs/gettingstarted/linux-install.sh <https://github.com/ClusterHQ/flocker/blob/master/docs/gettingstarted/linux-install.sh>`_,
-   - the ``box_version`` in
-     `docs/gettingstarted/tutorial/Vagrantfile <https://github.com/ClusterHQ/flocker/blob/master/docs/gettingstarted/tutorial/Vagrantfile>`_,
-   - `docs/gettingstarted/installation.rst <https://github.com/ClusterHQ/flocker/blob/master/docs/gettingstarted/installation.rst>`_ (including the sample command output) and
-
-   Commit the changes:
+#. Update the version numbers in the install and Vagrant scripts:
 
    .. code-block:: console
 
-      $ git commit -am "Bumped version numbers"
-
-   .. This should be automated. See https://clusterhq.atlassian.net/browse/FLOC-1038
+      envsubst < docs/gettingstarted/linux-install.sh.template > docs/gettingstarted/linux-install.sh
+      envsubst < docs/gettingstarted/tutorial/Vagrantfile.template > docs/gettingstarted/tutorial/Vagrantfile
```

Comment:
Perhaps, although is Python better than a shell script? Note that Linux (or at least Richard's Fedora machine and the Flocker dev environment) have `envsubst` already. We have been discussing recently not supporting OS X as an environment for doing this process and as I'm currently the only one on OS X I'm not sure that the dependency is a problem.